SmartDeviceLink/public/SDLImageResolution.m

```diff
@@ -39,6 +46,53 @@ - (void)setResolutionHeight:(NSNumber<SDLInt> *)resolutionHeight {
     return [self.store sdl_objectForName:SDLRPCParameterNameResolutionHeight ofClass:NSNumber.class error:&error];
 }
 
+- (CGSize)makeSize {
+    return CGSizeMake(self.resolutionWidth.floatValue, self.resolutionHeight.floatValue);
+}
+
+- (float)normalizedAspectRatio {
+    const float width = self.resolutionWidth.floatValue;
+    const float height = self.resolutionHeight.floatValue;
+    return (0 == width || 0 == height) ? 0 : fmaxf(width, height)/fminf(width, height);
+}
+
+- (SDLImageResolutionKind)kind {
+    if (!self.resolutionHeight || !self.resolutionWidth) {
+        return SDLImageResolutionKindUndefined;
+    }
+    const float ratio = self.resolutionWidth.floatValue/self.resolutionHeight.floatValue;
+    const float ratio2 = ratio * ratio;
+    const float tolerance = 0.001f;
+    if (ratio2 < 1.0 - tolerance) {
+        return SDLImageResolutionKindPortrait;
+    }
+    if (ratio2 > 1.0 + tolerance) {
+        return SDLImageResolutionKindLandscape;
+    }
+    return SDLImageResolutionKindSquare;
+}
+
+#define EQUAL_NUM(property) ((nil == self.property && nil == res2.property) ? YES : [self.property isEqualToNumber:res2.property])
```

Comment:
```suggestion
#define EQUAL_NUM(property) ((self.property == nil && res2.property == nil) ? YES : [self.property isEqualToNumber:res2.property])
```

It looks like this its defined and used multiple times

Reply:
It might be defined a few times though every time it is undefined after being used